litmus-portal/frontend/src/pages/Community/styles.ts

```diff
@@ -0,0 +1,85 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+// Community Component Styling
+const useStyles = makeStyles((theme) => ({
+  root: {
+    height: '100vh',
+    flexGrow: 1,
+    display: 'flex',
+    fontFamily: 'Ubuntu',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+  },
+  mainHeader: {
+    color: theme.palette.text.primary,
+    fontSize: '2rem',
+    marginBottom: theme.spacing(6),
+  },
+  LitmusAnalyticsBlock: {
+    margin: '2em 0',
+  },
+  header2: {
+    fontSize: '1.5rem',
```

Comment:
When you have normal text try using the Typography component, that will give us a consistent as well as responsive font size.

Reply:
I'm actually using Typography for wrapping the texts, in places where you see Header2 that is a custom component which wraps the texts with Typography but with a specific style